apis/experimentapi/experimentapi.yaml

```diff
@@ -0,0 +1,661 @@
+                    msgid: null
+                    err: INTERNAL_ERROR
+                    status: SERVER_ERROR
+                    errmsg: 'Process failed,please try again later.'
+                  responseCode: SERVER_ERROR
+                  result: {}
+        deprecated: false
+  security:
+    - bearerAuth: []      
+    
+  components:
+    securitySchemes:
+      bearerAuth:
+        type: http
+        scheme: bearer
+        bearerFormat: JWT
+    schemas:
+      Params:
+        title: Params
+        required:
+          - resmsgid
+          - err
+          - status
+        type: object
+        properties:
+          resmsgid:
+            type: string
+            nullable: true
+          err:
+            type: string
+          status:
+            type: string
+      ErrorParams:
+        title: ErrorParams
+        required:
+          - resmsgid
+          - err
+          - status
+          - errorMsg
+        type: object
+        properties:
+          resmsgid:
+            type: string
+          err:
+            type: string
+          status:
+            type: string 
+          errorMsg:
+            type: object
+            properties:
+              status:
+                type: string
+      CreateExperiment:
+        title: CreateExperiment
+        required:
+          - request
+        type: object
+        properties:
+          request:
+            $ref: '#/components/schemas/Experiment-Request'
+        example:
+          request:
+            expId: U1234
+            name: USER_ORG
+            description: Experiment to get users with orgname as sunbird to explore page 
+            createdBy: User1
+            criteria: 
+              type: user
+              filters:
+                organisations.orgName: [sunbird]
+            data:
+              startDate: '2020-10-11'
+              endDate: '2020-10-20'
+              key: 'ASDSDFDK23SDFKJDK'
+              client: app
+              
+      Experiment-Request:
+        title: Experiment-Request
+        required:
+         - expId
+         - name
+         - createdBy
+         - description
+         - criteria
+         - data
+        type: object
+        properties:
+          expId:
+           type: string
+           description: Unique id for experiment
+          name:
+           type: string
+           description: name of the experiment
+          createdBy:
+           type: string
+           description: name of the user who created the experiment
+          description:
+           type: string
+           description: Short description about the experiment
+          criteria:
+            $ref: '#/components/schemas/Criteria'
+          data:
+            $ref: '#/components/schemas/Data'
+      Criteria:
+       title: Criteria
+       required:
+         - type
+         - filters
+       properties:
+         type:
+           type: string
+           description: Experiment type - either user or device
```

Comment:
Type of the Experiment. Currently supported types are User and Device. Do we also support one in every 'n' Users or Device using modulus as part of this API?